es-ds-docs/nuxt.config.ts

```diff
@@ -10,4 +10,27 @@ export default defineNuxtConfig({
 
     // https://nuxt.com/docs/getting-started/deployment#static-hosting
     ssr: true,
+
+    // Per suggestion from https://github.com/nuxt/nuxt/issues/26783#issuecomment-2058966800
+    // See also:
+    // - https://github.com/nuxt/nuxt/issues/24196#issuecomment-1853870504
+    vite: {
+        optimizeDeps: {
+            include: [
```

Comment:
This list would eventually include all primevue components we're wrapping, and could be moved into the `es-ds-components` later.